ocp4/profiles/moderate.profile

```diff
@@ -230,19 +231,19 @@ selections:
     - selinux_policytype
 
     ### Application Whitelisting (RHEL 8)
-    - package_fapolicyd_installed
-    - service_fapolicyd_enabled
+    #- package_fapolicyd_installed
+    #- service_fapolicyd_enabled
 
     ### Enable the Hardware RNG Entropy Gatherer Service
-    - service_rngd_enabled
+    #- service_rngd_enabled
 
     ### Configure SSSD
     - sssd_run_as_sssd_user
 
     ### Configure USBGuard
-    - service_usbguard_enabled
-    - configure_usbguard_auditbackend
-    - usbguard_allow_hid_and_hub
+    #- service_usbguard_enabled
+    #- configure_usbguard_auditbackend
+    #- usbguard_allow_hid_and_hub
```

Comment:
These should already be approved for usage. Don't think that we should remove.

Reply:
They can be re-enabled once the packages ship in CoreOS.